Test/README.md

```diff
@@ -0,0 +1,57 @@
+# Test Automation
+
+## Introduction
+
+The following frameworks and packages are used for testing
+
+- [Playwright](https://playwright.dev/)
+
+### Notes:
+- Playwright will output screenshots and videos to help debugging test failures*
+
+- Debugging tests is made easy using the Playwright logger. Please refer to the `logger` block within the [Playwright Config](./playwright.config.js) and the [following documentation](https://playwright.dev/docs/api/class-logger) for more details
+
+## Prerequisites
+
+`node v18.16.x`
+
+`.env` file at the root of this directory with the following variables. Values can be directly copied from the Playwright Magento 2 Test Site 1password entry. See [example](env.example) file
+
+```
+KLAVIYO_PRIVATE_KEY=pk_...
+KLAVIYO_V3_URL=a.klaviyo.com/api
+METRIC_ID_ATC=AAAAAA
+M2_BASE_URL=https://www.example.com
+M2_ADMIN_USERNAME=someone@example.com
+M2_ADMIN_PASSWORD=astrongpassword
+```
+where METRIC_ID_ATC is the metric ID for the "Added To Cart" event in Klaviyo.
+
+## Instructions
+From this directory
+- `npm install`
```

Comment:
Just ran these locally and we need to add the step of downloading the browsers with `npx playwright install` otherwise they will fail

Reply:
hm really? I did the same yesterday and didn't need to run that step